### PR TITLE
Add German translation

### PR DESCRIPTION
**Pull request by alexmyczko**

### CODE REVIEW

File: ColorPickers/German.lproj/StandardPicker.strings [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/***
+Swedish.lproj/StandardPicker.strings
```
Comment (ivucica) [inline ColorPickers/German.lproj/StandardPicker.strings]: This is not Swedish.

File: ColorPickers/German.lproj/StandardPicker.strings [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+updated by make_strings 2002-05-23 16:24:57 +0200
```
Comment (ivucica) [inline ColorPickers/German.lproj/StandardPicker.strings]: Please remove this date, it is clearly inaccurate.

Comment (ivucica) [inline ColorPickers/German.lproj/StandardPicker.strings]: This date is incorrect and the comment superfluous, please remove.

File: ColorPickers/German.lproj/StandardPicker.strings [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /***
-Swedish.lproj/StandardPicker.strings
-updated by make_strings 2002-05-23 16:24:57 +0200
+German.lproj/StandardPicker.strings
+updated by make_strings 2017-08-02 16:24:57 +0200
```
Comment (ivucica) [inline ColorPickers/German.lproj/StandardPicker.strings]: This has not been updated by make_strings which is why you can eliminate this.

**Comment by ivucica** [comment]
Per discussion in IRC, as you do not use Git on the command line, I will accept these patches, squash them and write a commit description myself.

**Comment by fredkiefer** [comment]
I don't think we need a copyright assignment for these few translations. Please go ahead an merge this nice little patch.

**Comment by ivucica** [comment]
I got a confirmation of copyright assignment.

**Pull request merged**